src/components/Forms/GuestAccountCreation.vue

```diff
@@ -129,10 +133,14 @@ export default {
 					if (result?.errors?.length > 0) {
 						throw result.errors;
 					}
-					const resetUrl = result?.data?.general?.startGuestAccountClaim;
+					let resetUrl = result?.data?.general?.startGuestAccountClaim;
```

Comment:
Does this URL ever have other query parameters?

Reply:
This URL is coming from BE response and only query param we have is `login_hint=isGuest`

Reply:
Right, but below we are adding the new query string with `?`, can there be another query string there? If so we'll need `&`.

Reply:
Agreed, thanks clarifying.

Reply:
Yeah, mutation exclusive for guest account claim, always returning `?login_hint=isGuest`

Reply:
Should be fine if mutation could return `username` instead of passing across different components and query params... worth a try in the future.